src/DetailsView/components/left-nav/left-nav-icon.tsx

```diff
@@ -13,14 +13,14 @@ export const LeftNavStatusIcon = NamedFC<LeftNavIconProps>('LeftNavStatusIcon',
     const { item } = props;
 
     return (
-        <div>
+        <span>
```

Comment:
I'm wondering if this `<span>` is actually needed here

Reply:
Looks like not! updated